### ISSUE TITLE
[BUG] XML schema 1.4 `vulnerabilityType.affects.target.ref` has no type

### ISSUE THREAD

**Issue by jkowalleck** [CycloneDX/specification#182]
In the XML schema 1.4 I found the definition `vulnerabilityType.affects.target.ref`  had no `type` set.
see https://github.com/CycloneDX/specification/blob/ef71717ae0ecb564c0b4c9536d6e9e57e35f2e69/schema/bom-1.4.xsd#L1972 

from the corresponding JSON schema, and from the supplied annotation, I assume it is expected to be 
`type="bom:refType"`

**Comment by stevespringett** [comment]
Good catch. It defaults to string so technically it isn't causing issues, but this certainly needs to be fixed for v1.5. Thanks for catching this.

**Comment by jkowalleck** [comment]
The `bom:refType` is just an alias of `xs:string` with an annotation.
This could be changed in 1.4 without even a notice to the users. Therefore, I propose #183 to be merged without any version bump.

**Comment by jkowalleck** [comment]
do we not want to have this forward-merged to 1.5? 
@stevespringett
